mlflow/pyfunc/__init__.py

```diff
@@ -486,6 +486,75 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl, predict_fn=predict_fn)
 
 
+class _ServedPyFuncModel(PyFuncModel):
+    def __init__(self, model_meta: Model, client: Any, server_pid: int):
+        super().__init__(model_meta=model_meta, model_impl=client, predict_fn="invoke")
+        self._client = client
+        self._server_pid = server_pid
+
+    def predict(self, data):
+        result = self._client.invoke(data)
+        if isinstance(result, pandas.DataFrame):
+            result = result[result.columns[0]]
+        return result
+
+    @property
+    def pid(self):
+        if self._server_pid is None:
+            raise MlflowException("Served PyFunc Model is missing server process ID.")
+        return self._server_pid
+
+
+def _load_model_or_server(model_uri: str, env_manager: str):
+    """
+    :param model_uri: The uri of the model.
+    :param env_manager: The env_manager to load the model
+    :return: A tuple containing, in order: the PID of the model server (for non-local env managers),
+             and the loaded PyFuncModel.
```

Comment:
Can we update the return docstring and add a small docstring sentence indicating what this method does?